metals/src/main/scala/scala/meta/internal/builds/BuildTools.scala

```diff
@@ -98,6 +104,21 @@ final class BuildTools(
     else if (isGradle) GradleBuildTool.isGradleRelatedPath(workspace, path)
     else if (isMaven) MavenBuildTool.isMavenRelatedPath(workspace, path)
     else if (isMill) MillBuildTool.isMillRelatedPath(workspace, path)
+    else if (isPants) PantsBuildTool.isPantsRelatedPath(workspace, path)
     else false
   }
 }
+
+object BuildTools {
+  def all: List[BuildTool] = {
```

Comment:
This is only used in tests and there is an already existing `all` method. Could we remove it?